superset-frontend/src/components/Datasource/DatasourceEditor.jsx

```diff
@@ -698,6 +706,23 @@ class DatasourceEditor extends PureComponent {
     this.validate(this.onChange);
   }
 
+  async onQueryRun() {
+    this.props.runQuery({
+      client_id: this.props.clientId, // How this is generated?
+      database_id: this.state.datasource.database.id,
+      json: true,
+      runAsync: false,
+      catalog: this.state.datasource.catalog,
+      schema: this.state.datasource.schema,
+      sql: this.state.datasource.sql,
+      tmp_table_name: '',
+      select_as_cta: false,
+      ctas_method: 'TABLE',
+      queryLimit: 1000,
```

Comment:
I feel 100 rows here is more than enought, since the user probably just want to see a few samples? What's the use case for why this feature is being implemented?
```suggestion
      queryLimit: 100,
```

Reply:
Solved: I lowered to 25 according with design definition